node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
Is there a rationale for adding this? I don't see the problem with `if (led.value) ...`, is it really a good idea to start adding a bunch of aliases? Also, why is there so many ways to write a value? `high`, `low`, `on` (which also conflicts with being an EventEmitter), `off`, `toggle`, `write`.

How about just having `.value` and `.update(newValue, cb)`? Or do we need to maintain backwards compatibility?